tests/unit/DissolutionReviewConfirm.spec.ts

```diff
@@ -53,6 +53,8 @@ for (const test of reviewConfirmTestCases) {
 
       // verify page content
       expect(wrapper.find('h2').text()).toBe('Review and Confirm')
+
+      wrapper.destroy()
```

Comment:
Doing this for every test in this file saved about 100 MB of heap memory when running the entire test suites.

To verify heap usage I changed the script to this (in package.json):
```
"test:unit": "vue-cli-service test:unit --testPathPattern --runInBand --logHeapUsage",
```